tests/test_tools.py

```diff
@@ -34,29 +34,19 @@ def test_add_members_to_group(self, client):
         posted_group = client.add_members_to_group(posted_group, ['test_subject_y1@mail.com', 'test_subject_y2@mail.com'])
         assert posted_group
         assert len(posted_group.members) == 3
-        assert 'test_subject_x@mail.com' in posted_group.members
-        assert 'test_subject_y1@mail.com' in posted_group.members
-        assert 'test_subject_y2@mail.com' in posted_group.members
+        assert all(list(map(lambda email: email in posted_group.members, ['test_subject_x@mail.com', 'test_subject_y1@mail.com', 'test_subject_y2@mail.com'])))
```

Comment:
is there a change in this PR making these tests fail? why are you changing them?

Reply:
I prefer to keep the tests as simple as possible. Asserting each member individually is not a problem, it is clear for the users what the test is doing. 

If the test fails it is easier to know the reason. 

Also this is PR doesn't change any of this behavior, this change shouldn't be here.

Reply:
Reverted to original asserts.